### ISSUE TITLE
Menu bug in new version of Firefox

### ISSUE THREAD

**Issue by xmha97** [MrOtherGuy/firefox-csshacks#322]
Menu bug in new version of Firefox

![image](https://github.com/MrOtherGuy/firefox-csshacks/assets/23485114/179899e7-d3a2-408c-96fa-58503dad057d)

```css
/* Import your desired components first */

@import url(chrome/compact_extensions_panel.css);
@import url(chrome/drag_window_from_urlbar.css);
@import url(chrome/floating_findbar_on_top.css);
@import url(chrome/hide_tabs_toolbar.css);
@import url(chrome/iconized_main_menu.css);
@import url(chrome/privatemode_indicator_as_menu_button.css);
@import url(chrome/status_inside_urlbar.css);
@import url(chrome/window_control_placeholder_support.css);

/* Apply your custom modifications after imports */

/* Hide sidebar header */
#sidebar-header {
    display: none;
}

/* Hide sidebar on Fullscreen */
#main-window[sizemode="fullscreen"] #sidebar-box {
    display: none;
}

/* Hide forward button */
#forward-button {
  display: none !important;
}
```

**Comment by MrOtherGuy** [comment]
Seems I forgot hide_tabs_toolbar.css while adding workarounds for Firefox 119 in #315

This should be fixed in faf9766

**Comment by xmha97** [comment]
Thanks 🙏
